README.md

```diff
@@ -0,0 +1,115 @@
+<a name="readme-top"></a>
+
+Tasting Commons Global Summint, Check on this Website Our Schedule For May 11 this 2023
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+
+# 📖 Mobile LandPage <a name="about-project"></a>
+
+Schedule For Tasting Event on Venezuela, Miranda.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+HTML, CSS AND JS
+
+### Key Features <a name="key-features"></a>
+
+ Responsive Web Design 
+ DropDown Menu 
+ Displayed Menu Of Foods 
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## Live Demo
+
+[Live-Demo](https://gersonhiggins.github.io/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+Just to get information ab0ut my work
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+Visual Studio Code, Or another coding program.
+Git App
+Use this code to copy the project: https://github.com/gersonhiggins/capstone_project
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 👥 Authors <a name="authors"></a>
+
+
+👤 Gerson Higgins
+
+- GitHub: [@gitgersonhiggins](https://github.com/gersonhiggins)
+
+Template By Cindy Shin On Behance
+[CindyShin](https://www.behance.net/adagio07) - Cyndy's Profile.
+[ProjectTemplate](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) - Project Template.
```

Comment:
- [ ] Kindly use this line in the `Acknowledgments` section this is not an author so kindly remove this line from here and add in the `Acknowledgments` section in `line no.106` :slightly_smiling_face: